src/box/iproto_features.h

```diff
@@ -77,11 +77,19 @@ extern "C" {
 	 * tuple formats are received in IPROTO_TUPLE_FORMATS field.
 	 */								\
 	_(CALL_ARG_TUPLE_EXTENSION, 9)					\
+	/**
+	 *_(FETCH_SNAPSHOT_CHECKPOINT_CURSOR, 10) - only in EE:
+	 * IPROTO_IS_CHECKPOINT_JOIN, IPROTO_CHECKPOINT_VCLOCK and
+	 * IRPOTO_CHECKPOINT_LSN in FETCH_SNAPSHOT.
+	 */								\
```

Comment:
We do need to define the feature bit unconditionally and show it in `box.iproto.feature`. We just shouldn't send it in reply to `IPROTO_ID` or show it in `box.iproto.protocol_features`.

Reply:
Done